corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -62,7 +62,7 @@ def handle(self, **options):
             if self.local:
                 dest = self._apps_path('hqwebapp', 'js', bootstrap_version, 'requirejs_config.js')
                 copyfile(filename, dest)
-                logger.info(f"Copied {bootstrap_version}/requirejs_config.js back into {self._relative(dest)}")
+                logger.info(f"Copied {bootstrap_version}/requirejs_config.js back into {os.path.relpath(dest)}")
```

Comment:
hmm, it's certainly possible. It'll eventually fail because the path to `r.js` is relative, though that isn't deliberate. I'm inclined to either leave it alone or check the current directory and fail hard if it isn't in the root. Does that seem messy?

Reply:
I prefer tools that have minimal runtime context assumptions. But I understand if that seems like a nice to have and is out of scope for this PR.

Reply:
Cool, I will leave it for now and keep that in mind.